software/examples/snap_example_ddr.c

```diff
@@ -40,7 +40,7 @@
 #define	KILO_BYTE		(1024ull)
 #define	MEGA_BYTE		(1024 * KILO_BYTE)
 #define	GIGA_BYTE		(1024 * MEGA_BYTE)
-#define DDR_MEM_SIZE		(4 * GIGA_BYTE)	/* Default End of FPGA Ram */
+#define DDR_MEM_SIZE		(8 * GIGA_BYTE)	/* Default End of FPGA Ram */
```

Comment:
I wonder if we should have card specific include files or a general include snap_card_support.h or alike where we define card individual things. Inclusion or ifdefing for the appropriate card included.

-DSNAP_CARD=$(CONFIG_SNAP_CARD)
...

For now keep it.